components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -573,6 +573,45 @@ func (backend ES2Backend) getFiltersQuery(filters map[string][]string, latestOnl
 	boolQuery := elastic.NewBoolQuery()
 	boolQuery = boolQuery.Must(typeQuery)
 
+	filterTypes := []string{"environment", "organization", "chef_server", "chef_tags",
+		"policy_group", "policy_name", "status", "node_name", "platform", "role", "recipe",
+		"inspec_version"}
+
+	for _, filterType := range filterTypes {
+		ESFieldName := backend.getESFieldName(filterType)
+		termQuery := backend.newTermQueryFromFilter(ESFieldName, filters[filterType])
+		boolQuery = boolQuery.Must(termQuery)
+	}
+
+	if len(filters["control_name"]) > 0 {
+		termQuery := backend.newNestedTermQueryFromFilter("profiles.controls.title", "profiles.controls",
+			filters["control_name"])
+		boolQuery = boolQuery.Must(termQuery)
+	}
+
+	if len(filters["profile_name"]) > 0 {
+		termQuery := backend.newNestedTermQueryFromFilter("profiles.title", "profiles", filters["profile_name"])
+		boolQuery = boolQuery.Must(termQuery)
+	}
```

Comment:
'control_name' and 'profile_name' terms are special cases because they are nested terms. Nested terms in an array.  See the `newNestedTermQueryFromFilter` function.